src/services/queue/BaseQueue.ts

```diff
@@ -82,4 +82,15 @@ export default abstract class BaseQueue implements IQueue {
 
     return customArray;
   }
+
+  formatDuration (duration: number): String {
+    // :param duration: miliseconds
+    // We can use Moment.js library in the future, if needed
```

Comment:
It's fine like this for now. Moment.js is in legacy mode, so we should use either Luxon.js or plain JS (regex).

Reply:
👍